cdds/cdds/tests/test_prepare/test_command_line.py

```diff
@@ -179,9 +179,11 @@ def _main(self, request, use_proc_dir, output_dir, reconfigure=False):
         request.inventory.inventory_check = False
         request.data.variable_list_file = self.variable_list
         request.write(self.request_path)
+        plugin_id = request.metadata.model_id.split('-')[0]
```

Comment:
I think this will disappear following the arguments above, but there may be cases where this assumption that everything before the first `-` is not valid.